public/static/js/tour-controller.js

```diff
@@ -64,6 +64,9 @@ dlangTourApp.controller('DlangTourAppCtrl',
 			},
 			'Ctrl-R': function(cm) {
 				$scope.$apply('reset()');
+			},
+			'Ctrl-F': function(cm) {
```

Comment:
Okay, on a first attempt [cfp hotkeys](https://github.com/chieffancypants/angular-hotkeys) and the codemirror hotkeys didn't seem to support this, but "ALT-F" worked fine.
Btw there seems to be non-universal combination for formatting anyways, so I guess we are quite free in picking one, e.g:

- IntelliJ: Ctrl+Alt+L.
- Eclipse: Ctrl + Shift + F
- Visual studio is [entirely different](https://stackoverflow.com/questions/4942113/format-code-shortcut-for-visual-studio).